### ISSUE TITLE
Sort by type and tag at the same time

### ISSUE THREAD

**Issue by underr** [tommy351/ehreader-android#24]
Like "show only mangas with the 'yuri' tag".

I didn't find a way to do this.

**Comment by zertap** [comment]
Indeed. It is quite hard to find things with this. Would be nice if the search would be improved. Also the "minimum rating" is something I use often. And somewhat related to this issue, the app could also support tag flagging.
